docs/task_refs/TODO_37_15_12_91.Popen_mock.md

```diff
@@ -1,5 +1,7 @@
 
-TODO_37_15_12_91: To support testing with external command invocation, implement Popen mock:
-*   Ability to specify calls which should be intercepted.
-*   Ability to specify calls which should be passed through.
-*   Raise on any unspecified calls. 
+TODO_37_15_12_91: Add tests with config-only plugin which starts external process with Popen, but mock Popen by PopenMock to verify that.
+
+To support testing with external command invocation, implement Popen mock:
+*   DONE: Ability to specify calls which should be intercepted.
+*   TODO: Ability to specify calls which should be passed through.
+*   DONE: Raise on any unspecified calls.
```

Comment:
Finalize PopenMock to use func call (with list of args) to configure it instead of passing `dict` with mandatory `tuple`-s as keys.